src/partcad/scene.py

```
@@ -1,15 +1,45 @@
+from .assembly import Assembly
 
-from .. import partcad as pc
 
+@telemetry.instrument()
+class Scene(Assembly):
+    path: typing.Optional[str] = None
 
-class Scene(pc.Assembly):
-    def __init__(self):
-        super().__init__()
+    def __init__(self, project_name: str, config: dict = {}):
```

Ruff (0.16.2)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)